ide-support/revsaveasstandalone.livecodescript

```diff
@@ -311,6 +311,10 @@ command revDoSaveAsStandalone pStack, pFolder, pSettings, @xStandalonePaths
       revOutputDirectories pFolder & slash
       put the result into tOutputFolder
       if there is no folder tOutputFolder then
+         if there is a file tOutputFolder then
+            throw "Cannot create a folder" && quote & tOutputFolder & quote && \
+                  ". This name is already taken. Please choose a different name."
+         end if
          create folder tOutputFolder
```

Comment:
@livecodepanos I think the issue here is in `revOutputDirectories` when it is incrementing the count to find an unused folder it should check if there is a file too. Something like:

```
private command revOutputDirectories pFolder
   local tTarget,tDirectory,tPlatforms,tCount,tOSXCount
   revStandaloneProgress "Checking directories..."
   repeat while there is a folder (pFolder & sStandaloneSettingsA["name"]&tCount) or \
               there is a file (pFolder & sStandaloneSettingsA["name"]&tCount)
         add 1 to tCount
   end repeat
```
However `create folder tOutputFolder` could fail for a number of reasons so we should also check that using the following:

```
create folder tOutputFolder
if the result is not empty then
       throw format("Cannot create a folder '%s'", tOutputFolder)
end if
```